src/front/styles/petcard.css

```diff
@@ -0,0 +1,30 @@
+#cardpet {
```

Comment:
prefiero que uséis clases en lugar de ids. Luego puede ocasionar errores en la especificidad de CSS